apps/cowswap-frontend/src/legacy/hooks/useActivityDerivedState.ts

```diff
@@ -154,7 +154,7 @@ export function getActivityState({
   }
 
   if (isExpired) {
-    return isOrder ? 'expired' : 'failed'
+    return 'expired'
```

Comment:
To properly display the status in txs history:

<img width="832" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/46ff6720-762d-4445-9cb3-bb5beb1a16de">